app/products/calls/store/types/calls.ts

```diff
@@ -25,6 +25,7 @@ export type CallParticipant = {
     id: string;
     muted: boolean;
     isTalking: boolean;
+    raisedHand: number;
```

Comment:
why is this a number? what are the potential values? everywhere in the code this is being treated as a boolean.

Boolean(0) -> false
Boolean(!= 0) -> true

Reply:
It's a number because it's the timestamp of when the hand was raised, or 0 if it isn't. It's how it's coded in the server, and sent to the webapp and mobile.

Reply:
got it

Reply:
That's probably our strongest feature and one that Zoom hasn't yet addressed :smile: